examples/charts/file/area.py

```diff
@@ -1,6 +1,9 @@
 from collections import OrderedDict
 
-from bokeh._legacy_charts import Area, show, output_file
+from bokeh.charts import Area, show, vplot, output_file, defaults
+
+defaults.width = 400
+defaults.height = 400
 
 # create some example data
 xyvalues = OrderedDict(
```

Comment:
I really don't like `xyvalues` either. It looks unusual, it has too many syllables. I'd *really* like to call these something else, everywhere. Even just generic `data` would be better.